perses/tests/test_relative_point_mutation_setup.py

```diff
@@ -13,7 +13,7 @@ def test_PointMutationExecutor():
         ionic_strength=0.15 * unit.molar,
         flatten_torsions=True,
         flatten_exceptions=True,
-        conduct_endstate_validation=False,
+        conduct_endstate_validation=False
```

Comment:
Why remove the trailing comma here?

Reply:
and here https://github.com/choderalab/perses/pull/967/files#diff-f62f3ff16f52afacdf9163c3263cd309db5bf545f8d886c0eefd5720f8603917R35